lib/api/test/unit/validateBody.test.js

```diff
@@ -0,0 +1,110 @@
+const { assert } = require('chai');
+const validateBody = require('../../units/validateBody');
+
+describe.only('validateBody', () => {
```

Comment:
btw in case you rely on the CI passing under the PR, this runs only certain tests

Reply:
(FWIW, I used to commit .only() a lot to the point I wrote myself a git hook preventing me to commit if .only() was found in tests... but then I discovered I can ``npm test -- --grep=validateBody`` and I do that now most of the time)

Reply:
Yes, true. Thanks for spotting out. It's a WIP committed, will be removed.